### DISCUSSION TITLE
Can't find the right version of Ubuntu?

### DISCUSSION THREAD

**Discussion by nico-san** [NightscoutFoundation/xDrip#3440] [Q&A]
I am a total newbie!

Trying to start NightScout on a Google Cloud VM (using Chrome on a MacBook). Following step by step - https://navid200.github.io/xDrip/docs/Nightscout/VirtualMachine.html

I have followed the instructions for setting up a virtual machine several times and each time it has said the version of Ubuntu is incorrect. The set up menu options appear to have been updated since the guide article, and the Boot Disk menu does not provide options to choose any other versions of Ubuntu, tho afaik I am choosing the correct one. I'm not sure where to start in terms of troubleshooting. Thanks in advance for any recommendations.
![Screenshot 2024-04-28 at 22 24 46](https://github.com/NightscoutFoundation/xDrip/assets/8452112/f88bd208-ee2c-47ef-912e-ea532a00d9dc)

**Comment by Navid200** [comment]
Just a second.  it is possible Google has changed the images.  Just give me 10 minutes please.

**Comment by Navid200** [comment]
No, I didn't have a problem.  Looking at your screenshot, I can see that you have chosen a wrong image.  It looks like you haven't chosen the minimal type of image.
There is also a video clip at the bottom of the same page you have linked.

Please tell me if it is not resolved after you try again.

**Reply by nico-san** [reply]
Thanks, that seems to have been the issue!

**Reply by Navid200** [reply]
Great!

Let me know if there is ever any other issue or concern.
